jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l

```diff
@@ -140,8 +140,7 @@
      (if is-charging :charging :discharging)))
 
 
-(defun go-to-spot (name &optional (relative-coords (make-coords))
-                        &key (undock-rotate nil) (clear-costmap t))
+(defun go-to-spot (name &key (relative-pos nil) (relative-rot nil) (undock-rotate nil) (clear-costmap t))
```

Comment:
Are we ok to chance API?
Specially why you changed coordinates to pos/rot? 
since go-to-spot will be put in pr2eus level in future, please add documentation string